main.go

```diff
@@ -126,9 +126,12 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Ingress")
 		os.Exit(1)
 	}
-	if err = svcReconciler.SetupWithManager(ctx, mgr); err != nil {
-		setupLog.Error(err, "Unable to create controller", "controller", "Service")
-		os.Exit(1)
+	// Setup service reconciler only if AllowServiceType is set to true.
```

Comment:
Comment seems to mention the previous name for the feature gate or flag.

Reply:
fixed